services/housetables/src/main/resources/application.properties

```diff
@@ -13,4 +13,8 @@ management.endpoint.shutdown.enabled=true
 management.endpoint.prometheus.enabled=true
 management.endpoint.beans.enabled=true
 management.metrics.web.server.request.autotime.percentiles=0.5,0.75,0.9,0.95,0.99
-management.metrics.distribution.percentiles-histogram.http.server.requests=true
\ No newline at end of file
+management.metrics.distribution.percentiles-histogram.http.server.requests=true
+management.metrics.distribution.percentiles-histogram.jvm.gc.pause=true
```

Comment:
Do we have any other gc metrics that we can add?

Reply:
See this https://spring.io/blog/2018/03/16/micrometer-spring-boot-2-s-new-application-metrics-collector#what-do-i-get-out-of-the-box 
So we get a bunch of metrics by default since micrometer auto configure them. Things like `jvm.gc.pause` is called `meter Id` and springboot provides `management.metrics.distribution.percentiles-histogram.*` configuration so that these meter-id can be pre-aggregated in a histogram manner. 

`jvm.gc.pause: Measures garbage collection pause times` and 
`jvm.memory.used: Measures JVM memory usage` are the common jvm related ones. 

BTW changes in this file is to align with what we emitted in tables-service. Right now, If browsing through Jarvis, other than the customized metrics that we emitted in tables-service, you will see difference in the metrics emitted by both services.

Reply:
Sounds good. Thanks for the details. Should be good for now.